app/database_etl/etl_main.py

```diff
@@ -461,7 +474,7 @@ def check_filter_options(dashboard_source):
         # Check to see if the new options are equal to the curr hardcoded options
         if new_options != set(curr_filter_options[filter_type]):
             changed_filter_options[filter_type] = new_options
-            logger.info(new_options)
+            logging.info(new_options)
```

Comment:
Should this be `logging.info` or `logger.info`?

Reply:
ya it should be logging